### DISCUSSION TITLE
Publish release on pypi

### DISCUSSION THREAD

**Discussion by windelbouwman** [eProsima/Fast-DDS-python#292] [Ideas]
Are there any plans to release a python wheel to pypi? This would simplify installation on windows.

Ideas:
- Use ABI3, to reduce the amount of wheels required to build
- Use something like cibuildwheel

**Comment by universea** [comment]
Great idea!

**Comment by windelbouwman** [comment]
I created a pull request with an initial idea: https://github.com/eProsima/Fast-DDS-python/pull/179/files

**Comment by universea** [comment]
> I created a pull request with an initial idea: https://github.com/eProsima/Fast-DDS-python/pull/179/files

Cool!

**Comment by Lovely-XPP** [comment]
I created a repo for local build whl. Here is the binary for python3.12 with Windows: https://github.com/Lovely-XPP/Fast-DDS-python/releases
And Now I complime whl files for Python 3.8 - 3.12.
